tests/playwright-test/reporter-json.spec.ts

```diff
@@ -235,3 +235,18 @@ test('should add line in addition to file json without CI', async ({ runInlineTe
   expect(stripAnsi(result.output)).toContain('[1/1] a.test.js:6:7 › one');
   expect(fs.existsSync(testInfo.outputPath('a.json'))).toBeTruthy();
 });
+test('should have starting time in results', async ({ runInlineTest }, testInfo) => {
+  const result = await runInlineTest({
+    'a.test.js': `
+      const { test } = pwt;
+      test('math works!', async ({}) => {
+        expect(1 + 1).toBe(2);
+      });
+    `
+  },   { reporter: 'json' });
+  expect(result.exitCode).toBe(0);
+  if (result.report.suites[0].specs[0].tests[0].results[0].startTime)
```

Comment:
So you want to check if this field a) exists and b) can be parsed as the Date object from JSON and c) has reasonable value. Something like this:

```js
const startTime = result.report.suites[0].specs[0].tests[0].results[0].startTime;
expect(new Date(startTime).getTime()).toBeGreaterThan(new Date('1/1/2000').getTime());
```

It still allows startTime to be lax (not ISO format), but that's fine for the purpose of this test.

Reply:
The issue with this code is what I had earlier and why I wrote that mess with an if statement, 
![image](https://user-images.githubusercontent.com/70008218/187294335-6ee8d83f-6283-4ff4-9592-f39363703086.png)

Is it alright to leave an error like that?

Reply:
Your startTime is nullable, which does not matter in the scope of this test, you can always cast to non-nullable via appending !